### PR TITLE
[change] Added geocoding check #310

### LINKED ISSUE #310: [change] Add geocoding check in deployment script
Geocoding tests are converted in deployment checks via [this PR](https://github.com/openwisp/django-loci/pull/97)

We should add this checks to ensure geocoding works as expected

### PR DESCRIPTION
**Pull request by Aryamanz29**
Closes #310

### CODE REVIEW

File: tasks/django.yml [hunk 1]
```diff
@@ -104,6 +104,16 @@
 - name: start redis for migration
   meta: flush_handlers
 
+- name: run geocoding checks
+  notify: reload supervisor
```
Comment (codesankalp) [inline tasks/django.yml:108]:
```suggestion
  notify: reload supervisor
```
This can be removed

Comment (Aryamanz29) [reply tasks/django.yml:108]: Ok :+1:

File: defaults/main.yml [hunk 2]
```diff
@@ -38,6 +38,7 @@ openwisp2_time_zone: "UTC"
 openwisp2_context: {}
 openwisp2_allowed_hosts: []
 openwisp2_leaflet_config: {}
+openwisp2_geocoding_check: true
```
Comment (codesankalp) [inline defaults/main.yml:41]: Please update this variable in [README](https://github.com/openwisp/ansible-openwisp2#role-variables)

Comment (Aryamanz29) [reply defaults/main.yml:41]: Sure

**Review by nemesisdesign — APPROVED** [review]
Thanks @Aryamanz29 :+1:

**Pull request merged**